R/getTopTaxa.R

```diff
@@ -0,0 +1,88 @@
+#' Returns top N \dQuote{FeatureID}
+#'
+#' This function is will extract top user specified number of \dQuote{FeatureID}
+#' in a \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}} object.
+#'
+#' @param x A
+#'  \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}} object.
+#'
+#' @param top Numeric value, how many top taxa to return. Default return top five taxa.
+#'
+#' @param method Specify the method to determine top taxa. Either sum, mean or median.
+#'               Default is 'mean'
+#'
+#' @param abund_values a \code{character} value to select an
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
+#'
+#' @return A vector of the top N \dQuote{FeatureID}
+#' @name getTopTaxa
+#'
+#' @author
+#' Sudarshan A. Shetty
+#'
+#' @examples
+#' data(GlobalPatterns, package = "MicrobiomeExperiment")
+#'
+#' top_taxa <- getTopTaxa(GlobalPatterns,
+#'                        method="mean",
+#'                        top=5,
+#'                        abund_values="counts")
+#'
```

Comment:
if you add add a print command for top_taxa, this will make the pkgdown much more readable. Like this

```
#' top_taxa <- getTopTaxa(GlobalPatterns,
#'                        method="mean",
#'                        top=5,
#'                        abund_values="counts")
#' top_taxa
```